clients/python-wrapper/lakefs/config.py

```diff
@@ -45,12 +46,16 @@ class Credentials(LenientNamedTuple):
         access_key_id: str
         secret_access_key: str
 
-    configuration: Configuration
     server: Server
     credentials: Credentials
 
-    def __init__(self, **kwargs):
-        self.configuration = Configuration(**kwargs)
+    def __init__(self, verify_ssl: Optional[bool] = None, proxy: Optional[str] = None, **kwargs):
+        super().__init__(**kwargs)
+        if verify_ssl is not None:
+            self.verify_ssl = verify_ssl
+        if proxy is not None:
+            self.proxy = proxy
```

Comment:
```suggestion
        if verify_ssl:
            self.verify_ssl = verify_ssl
        if proxy:
            self.proxy = proxy
```

Reply:
@idanovo Thanks, note that this is a common mistake in python:
`if verify_ssl` will return False if `verify_ssl` is `None` or `False`.
In this case we want to assign the values only if they are not None (i.e. passed explicitly) whether they are True or False, and if not passed (or None) fallback to the client default settings.
In general the convention in python is to always use `is not None` to avoid logical mistakes when checking if value exists.